docs/content/manual/release-specific/v1.1.1/tolerations_setting.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Set Tolerations For System Components and User-deployed
```

Comment:
sugget to change to `Set Tolerations/PriorityClass For System Components`

Q: what do you mean `user-deployed`?

Reply:
Thank you. Fixed
`user-deployed` components are components created directly by user (i.g., Longhorn manager, driver deployer, and UI) as oppose to to system components are the components created by Longhorn (CSI components, IM pods, share manager pod, EI daemonset, backing-image-manager, cronjob)